pkg/logcli/query/tail.go

```diff
@@ -18,6 +22,15 @@ func (q *Query) TailQuery(delayFor int, c *client.Client, out output.LogOutput)
 		log.Fatalf("Tailing logs failed: %+v", err)
 	}
 
+	go func() {
+		stopChan := make(chan os.Signal, 1)
+		signal.Notify(stopChan, os.Interrupt, syscall.SIGTERM)
+		<-stopChan
+		if err := conn.WriteMessage(websocket.CloseMessage, websocket.FormatCloseMessage(websocket.CloseNormalClosure, "")); err != nil {
+			log.Println("Error closing websocket:", err)
+		}
```

Comment:
Let us add `os.Exit(0)` after this line since older builds won't handle this close message which makes cli program to continue running since we are handling those interrupts and waiting for the server to gracefully close the connection to break the loop down below.

Reply:
Good point about the older server builds. Since we are exiting here no need to introduce a change to the main loop anymore.